src/trivia/libraries/problems/problems.go

```diff
@@ -30,17 +31,27 @@ func GetProblems() ([]Problem, error) {
 func parseLines(lines [][]string) ([]Problem, error) {
 	ret := make([]Problem, len(lines))
 	for i, line := range lines {
+		if len(line) != 5 {
+			fmt.Printf("Wrong number of data columns, expected 5, received %d", len(line))
+			os.Exit(1)
```

Comment:
this should definitely just be an error return - yes the program will eventually exit because of it but that's not something parse should do

and the same below. You could just return an error with that message

Reply:
4752ed0